support-frontend/assets/components/subscriptionCheckouts/address/addressFieldsStore.js

```diff
@@ -110,6 +112,10 @@ const applyBillingAddressRules = (fields: FormFields, addressType: AddressType):
       fields.country === 'CA' ? `Please select a ${addressType} province/territory.` : `Please select a ${addressType} state.`,
     ),
   },
+  {
+    rule: checkpostCodeLength(fields.postCode),
+    error: formError('postCode', `Postcode for ${addressType} should not be longer than 20 characters.`),
```

Comment:
```suggestion
    error: formError('postCode', `Please enter a ${addressType} postcode no longer than 20 characters.`),
```
I would also consider moving the two postcode rules next to each other.  As on line 100-103 is another rule, for non empty postcodes.